lua/vex_library/modules/server/e2helperfuncs.lua

```diff
@@ -3,7 +3,7 @@
 
 local printf = vex.printf
 local isbool, isnumber, isstring, isentity, isangle, isvector, istable, IsColor, isfunction, getmetatable, pcall, IsValid = isbool, isnumber, isstring, isentity, isangle, isvector, istable, IsColor, isfunction, getmetatable, pcall, IsValid
-local string_upper, table_copy, table_IsEmpty, table_IsSequential = string.upper, table.Copy, table.IsEmpty, table.IsSequential
+local string_upper, table_copy, table_IsSequential = string.upper, table.Copy, table.IsEmpty, table.IsSequential
```

Comment:
Bug: The left hand side defines 3 variables and the right hand side has 4.
Notice, `table_IsSequential` is defined as `table.IsEmpty`.